tools/solar-only/setup.sh

```diff
@@ -150,6 +137,13 @@ if [ -f weather/weather411.conf ]; then
     docker restart weather411
 fi
 
+# Setup tesla-history
+if [ -z "${TZ}" ]; then
+    TZ="${CURRENT}"
+fi
+echo "Setup tesla-history..."
+docker exec -it -w /var/lib/tesla-history tesla-history python3 tesla-history.py --setup --timezone "${TZ}"
```

Comment:
I noticed that windows systems don't seem to run docker interactive mode (`exec -it`) very well.  I had to use `--tty` only for `verify.sh`.   I'll try this tonight on a windows system when I can.

Reply:
Yes, as I suspected.  This happens on my Windows 11 system:

```bash
$ docker exec -it pypowerwall sh
the input device is not a TTY.  If you are using mintty, try prefixing the command with 'winpty'
```

But it does work with winpty prefix:

```bash
$ winpty docker exec -it pypowerwall sh
/app $ uname
Linux
/app $
```

Reply:
I wish I knew a Windows guru that could help. I suspect we could just treat this with a condition to the script like this:

```bash
if type winpty > /dev/null; then
    winpty docker exec -it -w /var/lib/tesla-history tesla-history python3 tesla-history.py --setup --timezone "${TZ}"
else
    docker exec -it -w /var/lib/tesla-history tesla-history python3 tesla-history.py --setup --timezone "${TZ}"
fi
```